components/tabs/tab-mixin.js

```diff
@@ -12,6 +12,9 @@ export const TabMixin = superclass => class extends SkeletonMixin(superclass) {
 	static get properties() {
 		return {
 			selected: { type: Boolean, reflect: true },
+			// eslint-disable-next-line lit/no-native-attributes
+			role: { type: String, reflect: true },
```

Comment:
I'm guessing we need to reflect the role, given that `d2l-tab-internal` did. I suspect the purpose of doing so was to allow `d2l-tabs` to query them with `[role="tab"]`. I know in your other PR you are considering changing this.

However, does `tabIndex` need to be a reactive property?

Reply:
In a subsequent PR we do query for `tabIndex === 0` ([here](https://github.com/BrightspaceUI/core/pull/5450/files#diff-ce8cc2dbbd772bf9580235f279774d67c5a1c1f2ecb6f1b7afe0f8d3e67eef70R849)). I can remove this for now though until that change is applicable.